cassandra/cassandra_main.py

```diff
@@ -15,83 +15,40 @@
 import threading
 import argparse
 
+def bootstrap_sp(cfgfile_name):
```

Comment:
What is `sp`?

Reply:
"Single processing".  As contrasted with `mp` for multiprocessing.  I'm open to other names if you don't like that one.  There are, as they say, only two hard things in computer science:  cache invalidation, and naming things.

Reply:
The name is fine with me as long as the words "single processing" appear somewhere near the declaration or function call.

Reply:
Sounds reasonable.  Could you just add it wherever you feel it fits best?  I've got another branch checked out right now, and I'm juggling a bunch of WIP.